dev-util/ruff/ruff-0.3.7.ebuild

```diff
@@ -422,8 +422,13 @@ src_configure() {
 }
 
 src_compile() {
-	local -x CARGO_FEATURE_UNPREFIXED_MALLOC_ON_SUPPORTED_PLATFORMS=1
-	local -x JEMALLOC_OVERRIDE="${ESYSROOT}/usr/$(get_libdir)"/libjemalloc.so
+	if use elibc_musl || use elibc_Darwin || use elibc_bionic; then
+		# Gentoo bug #927338
+		local -x CARGO_FEATURE_UNPREFIXED_MALLOC_ON_SUPPORTED_PLATFORMS=0
```

Comment:
This should be the default so we don't need the if case, I think.

What about using `if use !elibc_musl && use !elibc_Darwin && use !elibc_bionic` instead?

Reply:
Ok, I've inverted the if statement